jmeter-custom-plugin/README.md

```diff
@@ -0,0 +1,33 @@
+---
+page_type: sample
+languages:
+- java
+products:
+- jmeter Custom Plugin.
```

Comment:
Please remove, as this is not an official product in the metadata system.

Reply:
Done

Reply:
Please remove the line for jmeter custom plugin. This is not an official product in the metadata system